---
layout: job
status: open
category: trainee
title: Postdoctoral Fellow (Genome Assembly)
subtitle:
image: 
pdf:
jobid: 1984382
---

The Barthel Laboratory is a cancer genomics lab focused on utilizing state-of-the-art genomic approaches to study the development and evolution of glial brain tumors (gliomas). The lab has a particular interest in the fundamental role of telomere dysfunction in glioma development, but is generally interested in all aspects of tumor evolution including but not limited to longitudinal studies of therapy response assessed by genomic analyses of tissue samples and liquid biopsies. We are looking for a talented, passionate and highly motivated postdoctoral fellow to work in the area of cancer genome evolution using long-read sequencing.

### Position Summary
The Postdoctoral Fellow will work on projects utilizing the latest technologies in long and short-read sequencing for analyses of structural variation and genome assembly. We are equipped with both the PacBio Revio and Oxford Nanopore Promethion long-read sequencers. Our wet lab is skilled in library preparation for long-read sequencing and Hi-C using standard short reads, ensuring a consistent supply of data to work with. The fellow will take part in the development of new tools for analyzing long reads and oversee the analysis of larger cohorts of samples.

### Requirements
- PhD in Bioinformatics, Computer Science, Bioengineering, Computational Biology or related field and 2+ years of experience in an applied research environment in biomedical data analysis
- Comfortable navigating Linux/UNIX based operating systems (eg. CentOS, Red Hat) and building shell scripts for data processing
- Confident operating high-performance cluster systems via schedulers such as SLURM or Torque/PBS
- Experience with scientific computing and data visualization tools such as R, Python or MATLAB and a good understanding of relevant statistics
- Exposure to biological software development in one or more programming languages, such as Python, C++, Java or related languages
- Experience building workflows using custom scripts and/or workflow management tools (eg. Nextflow, Snakemake, Common Workflow Language)
- Good coding practices utilizing versioning (eg. Git, Subversion) and writing detailed documentation
- Experience in biological data analysis and interpretation such as sequence alignment, data quality control, gene expression analysis, variant calling, phylogenetics
- Familiarity with various biomedical databases such as UCSC Genome Browser, Ensembl, TCGA/ICGC, GTex and other databases

### Location
We are located at the Translational Genomics Institute (TGen), a non-profit research institute located in Phoenix, Arizona that is set to the backdrop of the stunning Sonoran Desert. Phoenix is the 5th largest city in the nation, the fastest growing for 5 years in a row and ranked 5th in the Nation as an Emerging Life Science Market. Phoenix's unprecedented growth in the life sciences is fueled by over $3.25 billion in capital investment which has led to over 5 million square feet of new bioscience healthcare facilities constructed since 2019. At the heart of this growth is a 30-acre urban life sciences center - the Phoenix Biomedical Campus (PBC), at which TGen sits at the center.
Phoenix additionally boasts an incredibly comfortable lifestyle averaging over 300 sunny days a year, multiple mountain ranges in near proximity and some of the most magnificent national and state parks in the nation within driving range. The airport is ranked as one of the best in the country in terms of passenger satisfaction owing to fast moving lines at security checkpoints, and moreover is a mere 10-minute drive from TGen, making traveling to anywhere in the world a breeze.

### Application
Please direct all questions to barthel<span style="display:none">obfuscate</span>@barthel-lab.com. Application via the "Apply now" button on the left menu. Attach a letter describing your research interests and motivation to join our group. Include an academic CV detailing your past research experience, publication track record and (conference) presentations. We will start reviewing applications immediately and until the position is filled. 
